public/data/en/strings.json

```diff
@@ -1,6 +1,7 @@
 {
     "notRequired": "Not required for your state",
-    "newWindow": "Print your mail-in form in a new window",
+    "newWindow": "Go to your state's PDF form in a new tab",
```

Comment:
When testing I saw that some links on the first few pages where using this as the title and updated to be more generic